src/components/organisms/Form/FormSection/FormSection.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, Children } from 'react';
+import Heading from '../../../atoms/Heading/Heading';
+import Text from '../../../atoms/Text/Text';
+
+type FormSectionProps = {
+  title: string;
```

Comment:
I would like these to be optional. We don't use them in investments so it would prevent us from using it.